packages/qualys_vmdr/data_stream/asset_host_detection/manifest.yml

```
@@ -0,0 +1,80 @@
+title: Collect Asset Host Detection logs from Qualys VMDR platform.
+type: logs
+streams:
+  - input: cel
+    title: Asset Host Detection logs
+    description: Collect Asset Host Detection logs from Qualys VMDR platform.
+    template_path: input.yml.hbs
+    vars:
+      - name: url
+        type: text
+        title: URL
+        description: Base URL of the Qualys Server API.
+        default: https://qualysapi.qg1.apps.qualys.in/api/2.0/fo
+        required: true
+        show_user: true
+      - name: interval
+        type: text
+        title: Interval
+        description: "Interval between two REST API calls. User can choose interval as per their plan mentioned in [Qualys API Limits](https://www.qualys.com/docs/qualys-api-limits.pdf). NOTE: Supported units for this parameter are h/m/s."
+        default: 4h
+        multi: false
+        required: true
+        show_user: true
+      - name: batch_size
+        type: integer
+        title: Batch Size
+        description: "Batch size for the response of the Qualys Server API. This paramter specifies the truncation limit for the response. Specify 0 for no truncation limit. Note: No need to provide truncation_limit parameter in the Input Paramter text box."
```

```suggestion
        description: "Batch size for the response of the Qualys Server API. This parameter specifies the truncation limit for the response. Specify 0 for no truncation limit.
```

The "Note: No need to provide truncation_limit parameter in the Input Paramter text box." is confusing; it's not clear that anyone would worry about it; unless they already though that "Input Parameter" was the way to do this. Is there a reason this might be the case? Also sp. "Parameter".